### ISSUE TITLE
Publish API documentation

### ISSUE THREAD

**Issue by FeldrinH** [estnltk/estnltk#120]
Maybe I've missed something obvious, but I can't seem to find API documentation for the EstNLTK Python libraries. The documentation section of this repo lists tutorials, but I'm not looking for tutorials, I'm looking for searchable documentation of specific functions and classes.

**Comment by swenlaur** [comment]
Indeed, there is no single document that assembles the documentation of all API calls. The documentation for all function calls and classes is inside Pyhton docstrings, which are accessible from Jupyter nothebook. For instance call
`
from estnltk import Text
?Text
?Text.__init__
`
For some methods there no docstrings as the function name together with typing gives all the meaningful information.
  
To get all fields and methods of particular class use `dir` method to list all all options
`
dir(Test)
`
This together with tutorials should suffice for 99% cases. If you want to know go beyond that and ponder about very specific details of the methods just read the source code.

**Comment by FeldrinH** [comment]
Also the ability to link the documentation of specific functions/classes for sharing with others would be really useful.
